tron/kubernetes.py

```diff
@@ -123,7 +123,42 @@ def handle_event(self, event: Event) -> None:
         except Exception:
             self.log.exception(f"Unable to log event info for id={event_id}.")
 
-        # TODO(TRON-1611): actually transition to different states
+        if k8s_type == "pending":
+            pass
+        elif k8s_type == "running":
+            self.started()
+        elif k8s_type == "succeeded":
+            self.exited(0)
+        elif k8s_type == "failed":
+            self.exited(1)
+        elif k8s_type == "unknown":
+            self.log.warning("Kubernetes does not know anything about this task, it is UNKOWN")
```

Comment:
This is interesting, I feel like taskproc isn't submitting all of these `platform_type`s, we only seem to submit `task_event`s with:
* `killed`
* `finished`
* `FAILED` (uh oh, was this a typo?)
* `running`
* `unknown`

Are we sending the others (`pending`, `succeeded`, `failed` (I still think that's a typo)) from some other source?

Reply:
Nope. This was my mistake! I was misunderstood and was using pod phases instead of the ones emitted by TaskProc. Changed it.